### PR TITLE
Fix gpib addressing to remove redundent ::INSTR.

### PR DESCRIPTION
**Pull request by BrooksCampbell**
Breaking up logical chages from #358.

This branch removes a redundant ::INSTR we were adding to USB devices in the gpib server.

I don't know that this is always the correct thing to do, but it definitely is for the two USB scopes I've used:  Agilent DSA90804A scope and DSO-X2014A

### CODE REVIEW

File: gpib_server.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def refreshDevices(self):
                     elif addr.startswith('TCPIP'):
                         instName = addr
                     elif addr.startswith('USB'):
-                        instName = addr + '::INSTR'
+                        instName = addr
```
Comment (maffoo) [inline gpib_server.py]: Is the problem that `addr` already includes `::INSTR`, so adding this again is not needed? Or is the problem that the instrument identifier doesn't need the suffix `::INSTR` at all?

If we do this, then the three branches can all be collapsed into check:
```python
KNOWN_DEVICE_TYPES = ('GPIB', 'TCPIP', 'USB')
...
if not addr.startswith(KNOWN_DEVICE_TYPES):
    continue
instr = rm.get_instrument(addr)
...
```

Comment (maffoo) [inline gpib_server.py]: I learned about passing a tuple of strings to `startswith` in #205.

Comment (BrooksCampbell) [inline gpib_server.py]: I jut double checked and the the USB scope's address already ends in '::INSTR'.

Is "KNOWN_DEVICE_TYPES" a global?  If so, I think I made these changes correctly in the next commit.

Comment (maffoo) [inline gpib_server.py]: Yeah, I was imagining that tuple as being a module-level global, just like you did it.

File: gpib_server.py [hunk 2]
```diff
@@ -62,6 +62,7 @@
 ### END NODE INFO
 """
 
+KNOWN_DEVICE_TYPES = ('GPIB', 'TCPIP', 'USB')
 
 class GPIBBusServer(LabradServer):
```
Comment (maffoo) [inline gpib_server.py]: nit: two blank lines around top level definitions (https://google.github.io/styleguide/pyguide.html#Blank_Lines)

**Comment by maffoo** [comment]
One formatting nit, then LGTM.

**Pull request merged**